src/api/DataIntegrationApi.test.js

```diff
@@ -121,5 +139,23 @@ function createEntityAndAssociationData() {
     testApiShouldReturnPromise(apiToTest, validParams);
     testApiShouldUseCorrectAxiosInstance(apiToTest, validParams, DATA_INTEGRATION_API);
   });
+}
+
+function getEntityKeyIds() {
+  describe('getEntityKeyIds()', () => {
+
+    const apiToTest = DataIntegrationApi.getEntityKeyIds;
 
+    const validParams = [MOCK_ENTITY_KEYS];
+
+    const invalidParams = [INVALID_MOCK_ENTITY_KEYS];
```

Comment:
Due to the way the test helper function is written, using `INVALID_PARAMS` instead would cover more edge cases.

```
import { INVALID_PARAMS } from '../utils/testing/Invalid';
...
const invalidParams = [INVALID_PARAMS];
```